packages/vats/test/test-tokens.js

```diff
@@ -5,7 +5,7 @@ import test from 'ava';
 import { assertKeywordName } from '@agoric/zoe/src/cleanProposal.js';
 
 import { AssetKind } from '@agoric/ertp';
-import { Stable, Stake } from '@agoric/inter-protocol/src/tokens.js';
+import { Stable, Stake } from '@agoric/internal/src/tokens.js';
```

Comment:
ah. this PR isn't nearly as big as the scrollbar suggested.

"Unchanged files with check annotations" made it look much larger.